### PR TITLE
[Lesson 4: Middleware, REST Methods, and Postman] Code Assignment

### PR DESCRIPTION
**Pull request by yuliahar**
1. Task: https://learn.codethedream.org/ctd-node-lesson-3-fall2023-b/

2.  Coding Assignment: 

- [x] implement logger middleware, people router and controller, products router and controller
- [x] optional additional assignment: auth middleware

3. Done 10.17.2023 / deadline 10.24.2023

### CODE REVIEW

File: 02-express-tutorial/logger.js
Comment (leo-mutuku) [inline 02-express-tutorial/logger.js:1]: Loggger well defined

**Review by leo-mutuku — APPROVED** [review]
logger middleware well implimented

File: 02-express-tutorial/app.js
Comment (leo-mutuku) [inline 02-express-tutorial/app.js:1]: express.urlencoded() ,express.json() add , post method well implimented

**Review by leo-mutuku — APPROVED** [review]
get , post methods well implimented

File: 02-express-tutorial/controllers/people.js
Comment (leo-mutuku) [inline 02-express-tutorial/controllers/people.js:1]: Controllers well implimented

**Review by leo-mutuku — APPROVED** [review]

**Review by leo-mutuku — APPROVED** [review]
routes and controllers well implimented

**Review by leo-mutuku — APPROVED** [review]
Crude operations well implimented

**Review by leo-mutuku — APPROVED** [review]
Simple auth well implimented

File: 02-express-tutorial/logger.js
Comment (yuliahar) [reply 02-express-tutorial/logger.js:1]: Thanks.

**Pull request merged**